examples/k8s-overlay-ingress/network-api-fix.yaml

```diff
@@ -0,0 +1,75 @@
+#! START OF COMMENT SECTION
+#! ------------------------
+#! We're overlaying the apiVersion used by falco charts
+#! We need to do so because we're rolling at least
+#! v1.22+ of Kubernetes ( as of 211230 )
+#! And this > https://github.com/falcosecurity/charts/blob/d2c6b91ff7339a5108abe23208212eb53b535463/falcosidekick/templates/_helpers.tpl#L38
+#! does not seem to work with the version value that the K3s distribution
+#! uses. For more background see > https://github.com/falcosecurity/charts/issues/288
+#! ----------------------
+#! END OF COMMENT SECTION
+
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+
+#@overlay/match by=overlay.subset({"apiVersion":"networking.k8s.io/v1beta1"}),expects="1+"
+---
+apiVersion: networking.k8s.io/v1
+
+#! START OF COMMENT SECTION
+#! ------------------------
+#! As we overlay the apiVersion used
+#! we need to reflect that change as well
+#! So in the below we're ytt overlaying on
+#! the backend of the [INGRESS FOR THE FALCO WEB UI]
+#! ----------------------
+#! END OF COMMENT SECTION
```

Comment:
As someone not very familiar with the falco helm charts, it isn't clear why these resources need to be patched as well. There is only mention of the `apiVersion` in the issue linked. Can you give greater detail here on why this is necessary?